lib/Scheduling/SimplexSchedulers.cpp

```diff
@@ -1115,12 +1116,27 @@ void ModuloSimplexScheduler::scheduleOperation(Operation *n) {
   (void)fixedN, (void)enteredN;
 }
 
+unsigned int ModuloSimplexScheduler::resII() {
+  unsigned int resII = 1;
+  DenseMap<Problem::OperatorType, unsigned int> uses;
```

Comment:
Nits: Prefer `unsigned` instead of `unsigned int`, name this `resMinII`, and note that I used `SmallDenseMap` for operator-type-indexed maps elsewhere in this code (assumption is that the number of operator types is << number of operations).